tutorials/Tutorial_For_QA_Registration.ipynb

```diff
@@ -0,0 +1,377 @@
llsuhRufe6gHUR3wnNZiTmPy6mLsU4xvNCiyriULXl1WveCoYTf2b6EaVNSzOab854RPx3zu1Z8fjh+GACw48fIpoe0wFsUP82XR3+U5iaNxQaeLtF0Rvd/mUbIqKA3YYwYzoa9/bYMJrTO0NpCSudisugzZNRqQOmtsuxW2xWZyAC4RjnWPFdWjOZPgsTHmIHYpQGC17PaELxiuVxTOv5OLJtLyThDY9K4ZUv/YvLitbHgRhoMt6c+utearfl0QLV9UEq1zn+eM4acbJ7EGTAejiWMFVPp1Kr965weaLJdRlfchaGY39WU5bbbrcsocs3V9fcudC3HNrd+VNxejHj/kClG2ZR9uUdPmXFl4CcHNjsrAbchOlWufgJ0BPwcaGaxAOAvAJxwfwOAywD+l60eYEBAQEBAQEBAQEBAwK2AjpQra+0kgL9ljLkHwP3u1y9aa0+u2OavbsL4bgwsl8HpidM4b6iKiGGQOYPYDzUINsb4QmkVR6p2SWyNXkfyEV+3o0LS9WxUa3nN72s2IaskplfGCp6ZWqMmQgyubJ8fBA0VxGqLdZqxM/4zxahJwRILJUUutalnyS/FfH9e5Ks/eyM1qv3nNdgnFbmr9kGq2kzCc6wc5Vkzi0sZxyAFTyyTLG7FnvVZ5jL32uWmgSqib29GOJSTdRq0gyhH3K8Yb7HkYqYOWLJZ++w+dKVkqfNeno+uD/HnHd07EHCL4gf4Ev0C54Fs9U90nUDa4rXl6xNj15oh5rV2OKFivLu0G1HC9w/XyO5+PWJz35lZZ8awwGtvKp7y9t5Srg4ussbwQkbFZNyw7qFhef1aa/31ecKsbsbbC7LfR4uj/IUBZuuMJUcz/q66j3O7b4zjnrVOuVqrzqoTu2i3za6YcefRMpn8x+eojEl9B5aLxqU4K74NJzxPD/VSMazNc16fq5/zsU/zWLWSikuqv6zaqq/zmE9oq3wZ/GwZZSgu7LSskRiOhn19hgrpTeYU84xji7LIf76UK33Xh3AIADDXIPN/3vIeMm2mfexIsZpNtxWON435exkNBdyaKO/hPeOBVx7Ay5ZW5jKe6rG8XnTd6L5VmMJnu2jOqvZ3l+U8kqJ5Mb7ozVVksKPaIpm91MF1geoSdc8E4OfqribfeyFhXHkldslDK+b4PDhv+iMqMlKwpHJdAdXYUYz661dtYlQnKaVZyvt9+X14OeF5US3ajdjNV8DzpTWK1ljd6PZrNDUEl3rUTKjWKxvpcnHZZ+foVWqaDMf2NpYNtACgUqp4MxAdm9ZAUu+0bohM5GUI1ZEqc0mZDknEsQ2bYQyUeO4OHWAcCWuHOxOdpgUCANzD1CbVlwEBAQEBAQEBAQEBAXceOn64cqrV+0FdaFVCubX2h7d4XK8KtR9xblUXDLonyYR2gQqErEZlZaqc5TzKfRPMPpCtkdIzkJGJOJBT4Xg4fxhPl5mHLfZKjeNUQ6A8arFaA/kA5goyoUcrZJuLawqeHFY48ZyJzvj3A8D9EYVDMaNiUMfM2LIboGNvVA8gW3F9XgUV7+glhkqskG8gulKNasuHFmKsZmf7bb//XNkxS3WSy44YoDx1TfdaDZ/zLIZqOppetV81cRRzWDIlr4Sp2W+9RJavlPLSlGPP7tJuf4x3RZQ0q4mr/6pyf0M1p2BVD2CgxzV2/onAPN9uKO/iPNi9QIfJHtPjI6CYx96YjPO74ncBWK4bGsAApmu8LlWDsXeJbOjz2fMAgCMZa7Eu28uYmuc1va2Pyu1gy7UKcLn52vbZhC6CTTQ9CyrIsXK+YFw6G5/1f6u2eA1PNfg5vzv9uwDglfobar+wYp6LtX1D+gYAwJsHaFu8J6ej1hOLTwBgA2XVio5bqnHett3N9Xu7aF++s5us8djlMYy1WBshdl7x8kjB83IoI+s7ZafwVPwUP8twTotZVo2m4rVizkQxgWbGcykFe6jF16zge6TYF1GBnpjv03c0lfGcivFfSpwqHmX+/qF4tt/sBwBsH2K9hq6jgFscLj9n5MoI/vrlvw4A+GLPFwEARexcPV0bhZ4Wr5/Yxr6xrbIv1PB2xPBePBNR5bax9c18dS+TYmUjTkjNiS7DtUWEyLclkcJzwHBNopYtR1OnbhfwDYHVHkB12nsM5/A+QwVLLnkz+QwWMu5fmTlyJZaKpNeyKeOR/BEAy61T/Lxfa1nTXoPtzp2cRncUVHd2RHxNbeqdFn29k5tbSczXuMX3mtz42jZZqilzRjVw+jyd44FiwGf2PGBZy3m1YH2+1HO5RZdNedmp0MVFOcSalJ/Tmzo34a4uPPT3qdTv79/P3wWH4TsSHd0JjDHfBuCTAF4Gw85RAAfB2+dXb9roAgICAgICAgICAgICbhF0SrP9PIB/Y639340xcwC+F8AlAL8LOgm+LvHuI+/GE8+TYR3PyKoq914qi/JsZ80s4phMyB6QyZHiM5CSvd5ZJvPaiBuebVI/KjFVUl+kWIkFSU3q66PEzqqJrZptDhT8HGstLkTMoRYLpPoOjV/qy7aYjFUVVVwyq+unfOM+J8IovxlYZoPUq0M1EAcj1oacAnvsiP1aBaM/8Y9is9TD42DtoM8TTxr8HDVwVY8K1bMlzcTXrSl3O0tW17Hp3KrGAoY508Dya5q6RqKOSaqW+HnvKr8LPV1kFmu93E/tH/J1aokMteptBqoDni0MuP0Q/QC/25FfIYu899JeXCmx3mBbifPou5LvAgDETU6aJHMhsgt4oUZHrucnqVRdWeJ7HzJkKtV4c8JM4HxGBam3TkazSJ1zZ53XnNjkjSCHsWeiZ/gLpzDFJvZ9aD599dMAgPE649u6TT/XqrNap9ediYxvlit3z74q1aG797Ofznuy9wAAWu9pYWyWKtQL/53n53P1zwEAnm5S4Tt0gSrUPz7wjwEA37n9O/G1afYd/OoSubmpfGrV67ylapQg8Y281adHfa9U/yA1SYr9rJnFjpzvOdQ85E4LT8zxEmtVpYbXUENfi
```

Comment:
What does "and the slice is not cut to the brain." mean? Please make this sentence clearer

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/7b32b3d114ee8d7a65060fb0dc1fefa8393cb052

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/e0287c16348579b8c3f2c7a633444006ec75074b